sbin/jtreg.sh

```diff
@@ -92,7 +92,7 @@ settingUpEnvironmentVariablesForJTREG()
 runJtregViaMakeCommand()
 {
   echo "Running jtreg via make command (debug logs enabled)"
-  if [ -z $JTREG_TEST_SUBSETS }; then
+  if [ -z "$JTREG_TEST_SUBSETS" ]; then
     make test jobs=10 LOG=debug
```

Comment:
I *think* either is fine, although `${}` is easier to read if you're already in double quotes already I guess (e.g. `NEWVAR="${VAR}SOME OTHER STRING"`}.